api/src/routes/photos.js

```diff
@@ -0,0 +1,31 @@
+import express from "express";
+import { sharksList, catsList } from "../data/imagesList";
+import { shuffleArray } from "../utils/index";
+
+const router = express.Router();
+
+/* GET photo list */
+router.get("/photos", (req, res, next) => {
```

Comment:
What are some changes you could make to your api route design to make it "more RESTful"?